src/Groups/GAPGroups.jl

```diff
@@ -304,11 +308,13 @@ function (g::PermGroup)(L::AbstractVector{<:Base.Integer})
    throw(ArgumentError("the element does not embed in the group"))
 end
 
+(g::PermGroup)(L::AbstractVector{<:fmpz}) = g([Int(y) for y in collect(L)])
```

Comment:
Why use collect like this (see above)?

Reply:
I used `collect` to make it work with `StepRange{fmpz,fmpz}`. Without the `collect`, the code
> L = fmpz(3):fmpz(7)
> [Int(y) for y in L]

would return the error `ERROR: MethodError: no method matching Base.OneTo(::fmpz)`.